CodeCoverageQualityGate/CodeCoverageQualityService.ts

```diff
@@ -170,32 +171,32 @@ export class CodeCoverageQualityService {
         return coverageInfo;
     };
 
-    private getCoberturaReportOfCurrentBuild = async (): Promise<FileEntry | undefined> => {
+    private getMergedCoberturaReportOfCurrentBuild = async (): Promise<FileEntry | undefined> => {
         const currentBuildArtifact = await this.buildClient.getCoberturaArtifactOfBuildById(Number(this.buildId));
         if (!currentBuildArtifact) {
             return;
         }
-        const coberturaReports = await this.buildClient.getCoberturaReportsOfArtifact(currentBuildArtifact);
+        const coberturaReports = (await this.buildClient.getMergedCoberturaReportsOfArtifact(currentBuildArtifact)).filter((file) => file.name.includes(this.MERGED_COBERTURA_REPORT_NAME));
```

Comment:
`getMergedCoberturaReportsOfArtifact` will get that report with specific name. `filter` part is not needed here.